test/distributed/org/apache/cassandra/fuzz/topology/TopologyMixupTestBase.java

```diff
@@ -336,12 +353,15 @@ private EnumSet<TopologyChange> possibleTopologyChanges(State<S> state)
         // so up is enough to know the topology size
         int up = state.topologyHistory.up().length;
         int down = state.topologyHistory.down().length;
+        int[] upAndSafe = state.upAndSafe();
         int total = up + down;
         if (total < state.topologyHistory.maxNodes)
             possibleTopologyChanges.add(TopologyChange.AddNode);
-        if (up > state.topologyHistory.quorum())
+        if (upAndSafe.length > 0)
         {
-            if (up > TARGET_RF)
+            // can't remove the node if all nodes are CMS nodes
+            var nonCMS = Sets.difference(asSet(upAndSafe), asSet(state.cmsGroup));
+            if (!nonCMS.isEmpty())
```

Comment:
if the only nodes that exist are CMS nodes, avoid remove as its blocked...

Reply:
Nit: could probably fold `var` into `if`.

Reply:
Also, _technically_ we could resize CMS, but we don't have to do this now.